### ISSUE TITLE
value or initialValue prop

### ISSUE THREAD

**Issue by spruce-bruce** [synapsestudios/react-drop-n-crop#3]
It'd be nice to figure out how to implement a `value` prop so that this can be turned into a controlled component. Obviously that would require more information than just the cropped `File` object that is output by `onSave`, so this might not be a good idea. 

In order to implement a value prop onChange needs to be updated to return an object shaped something like this:

```
{
  originalImage: File object,
  croppedImage: File object,
  croppedDimensions: {x: int, y: int, width: int, height: int}
}
```

These values could be useful in other scenarios anyway (especially if we decide to do image processing on the server), but changes fundamentally the simplicity of this component.

At the very least we'll want to do an `initialValue` prop that takes a `File` object for use cases like re-cropping an image that's already been uploaded.

**Comment by chrisheninger** [comment]
Great idea, would love to chat through details... I like what you've proposed. 👏

**Comment by chrisheninger** [comment]
Fixed in #4
